mne/evoked.py

```diff
@@ -455,8 +455,9 @@ def plot_joint(self, times="peaks", title='', picks=None,
 
     @fill_doc
     def animate_topomap(self, ch_type=None, times=None, frame_rate=None,
-                        butterfly=False, blit=True, show=True, time_unit='s',
-                        sphere=None, *, image_interp=_INTERPOLATION_DEFAULT,
+                        vmin=None, vmax=None, butterfly=False, blit=True,
```

Comment:
typically we wouldn't add new params to the middle of the signature like this. After the `*` in the signature, they can go wherever makes sense, but before the `*` we implicitly guarantee a fixed order.

Reply:
Ah I wasn't aware of this convention, but it makes sense. I'll change it accordingly